recipes/swga/meta.yaml

```diff
@@ -8,22 +8,26 @@ package:
 source:
   url: https://github.com/eclarke/swga/archive/2f240cbb356d8c4b4130d9e442596d10227738ff.tar.gz
   sha256: 28f6971234453f8ba91997c3203e71e94a4de5e37f7bcfd01096deb1fcb952c0
+  patches:
+    - SortingCount.cpp.patch
 
 build:
-  skip: true # [not py27]
+  skip: true # [osx]
   entry_points:
     - swga = swga.main:main
   number: 1
```

Comment:
Please also increase the build number here.

Reply:
@johanneskoester we increased all build numbers already, so in theory all packages should have a bump already included.